src/webparts/laserficheAdminConfiguration/components/AddNewManageConfiguration/AddNewManageConfiguration.tsx

```diff
@@ -69,10 +68,9 @@ export default class AddNewManageConfiguration extends React.Component<IAddNewMa
       },
       shouldShowOpen: false, 
       shouldShowSelect: false,
-      shouldDisableSelect: false
+      shouldDisableSelect: false,
+      region: this.props.devMode ? 'a.clouddev.laserfiche.com' : 'laserfiche.com'
```

Comment:
use double quote in all .tsx files, or use single quote in all files (.tsx and .ts)

Reply:
Looking more into this, it seems to still follow typescript conventions, so should be single quote, I'll do that in a future PR